tests/parse/mod.rs

```diff
@@ -281,24 +281,69 @@ fn cond() -> LangResult<'static, ()> {
     let result = parse(input)?.content;
     let expected = vec![
         Cond(
-            vec![Name(ast::Name("x")).loc()].loc(),
-            vec![Name(ast::Name("y")).loc()].loc(),
+            Branch {
+                cond: vec![Name(ast::Name("x")).loc()].loc(),
+                body: vec![Name(ast::Name("y")).loc()].loc(),
+            },
+            vec![],
             vec![Name(ast::Name("z")).loc()].loc(),
-        )
-        .loc(),
+        ).loc(),
         Cond(
-            vec![Name(ast::Name("u")).loc(), Name(ast::Name("v")).loc()].loc(),
-            vec![Name(ast::Name("w")).loc(), Name(ast::Name("x")).loc()].loc(),
+            Branch {
+                cond: vec![Name(ast::Name("u")).loc(), Name(ast::Name("v")).loc()].loc(),
+                body: vec![Name(ast::Name("w")).loc(), Name(ast::Name("x")).loc()].loc(),
+            },
+            vec![],
             vec![Name(ast::Name("y")).loc(), Name(ast::Name("z")).loc()].loc(),
-        )
-        .loc(),
+        ).loc(),
     ];
 
     assert_eq!(expected[0], result[0], "simple blocks");
     assert_eq!(expected[1], result[1], "long blocks");
     Ok(())
 }
 
+// #[test]
```

Comment:
Also remove the commented test (but plz save it somewhere else, there is no way anyone is going to rewrite that huge AST :P)